src/test/java/example/Example.java

```diff
@@ -86,5 +94,21 @@ public void initialize(Bootstrap<ExampleConfiguration> bootstrap) {
         public void run(ExampleConfiguration configuration, Environment environment) throws Exception {
             environment.jersey().register(new ExampleResource());
         }
+
+        @VisibleForTesting
```

Comment:
this is the example, i want to call out that it's ONLY for testing purposes and shouldn't be used as part of the example - so don't copy it. i hope the `@VisibleForTesting` would flag that.